src/components/com_tjcertificate/administrator/models/certificates.php

```diff
@@ -81,18 +88,64 @@ protected function getListQuery()
 		// Initialize variables.
 		$db    = $this->getDbo();
 		$query = $db->getQuery(true);
-		$app = Factory::getApplication();
+		$app   = Factory::getApplication();
+		$user  = Factory::getUser();
 
 		$extension = Factory::getApplication()->input->get('extension', '', 'CMD');
 
 		$this->setState('filter.component', $extension);
 
+		// Filter by client
+		$client = $this->getState('filter.client');
+
 		// Create the base select statement.
 		$query->select(array('ci.*', 'ct.title', 'users.name as uname'));
 		$query->from($db->quoteName('#__tj_certificate_issue', 'ci'));
+
 		$query->join('LEFT', $db->quoteName('#__tj_certificate_templates', 'ct') .
 			' ON (' . $db->quoteName('ci.certificate_template_id') . ' = ' . $db->quoteName('ct.id') . ')');
-		$query->join('LEFT', $db->quoteName('#__users', 'users') . ' ON (' . $db->quoteName('ci.user_id') . ' = ' . $db->quoteName('users.id') . ')');
```

Comment:
Why changed to inner join..we want to see certificates of deleted users as well

Reply:
updated

Reply:
updated